haproxy/dockerdir/etc/haproxy-auto.cfg

```diff
@@ -0,0 +1,20 @@
+    global
+      maxconn 100
+      external-check
+      stats socket /var/run/haproxy.sock mode 600 expose-fd listeners level user
+
+    defaults
+      log global
+      mode tcp
+      retries 2
+      timeout client 4m
+      timeout connect 2m
+      timeout server 4m
+      timeout check 5s
```

Comment:
Could you please adjust defaults to ProxySQL defaults https://github.com/percona/percona-xtradb-cluster-operator/blob/master/e2e-tests/self-healing/compare/proxy-vars-80.sql

Reply:
done